src/main/java/io/vertx/core/http/impl/Http2ClientConnection.java

```diff
@@ -132,6 +133,14 @@ private void recycle() {
     listener.onRecycle(expired);
   }
 
+  @Override
+  public synchronized void onHeadersRead(ChannelHandlerContext ctx, int streamId, Http2Headers headers, int streamDependency, short weight, boolean exclusive, int padding, boolean endOfStream) throws Http2Exception {
```

Comment:
rather change `stream#handleHeaders` to be `handleHeaders(Http2Headers,StreamPriority,boolean)` and handle this the same way than `onHeadersRead(ChannelHandlerContext ctx, int streamId, Http2Headers headers, int padding, boolean endOfStream)` do